Cache/Requests.lean

```diff
@@ -44,20 +44,26 @@ def mkGetConfigContent (hashMap : IO.HashMap) : IO String := do
     -- A backslash preceding any other letter is ignored.
     -- ```
     -- If this becomes an issue we can implement the curl spec.
+
+    -- Note we append a '_' to the filenames here,
+    -- which `downloadFiles` then removes when the download is successful.
     pure $ acc ++ s!"url = {← mkFileURL fileName none}\n-o {
-      (IO.CACHEDIR / fileName).toString.quote}\n"
+      (IO.CACHEDIR / (fileName ++ "_")).toString.quote}\n"
```

Comment:
How about `.part` as suffix. I think other programs also use that, when downloading. It's a bit more descriptive.